airbyte-cdk/java/airbyte-cdk/db-sources/src/main/kotlin/io/airbyte/cdk/integrations/debezium/internals/DebeziumRecordIterator.kt

```diff
@@ -255,16 +255,8 @@ class DebeziumRecordIterator<T>(
     private fun heartbeatPosNotChanging(): Boolean {
         if (this.tsLastHeartbeat == null) {
             return false
-        } else if (!isTest() && receivedFirstRecord) {
```

Comment:
shall we also remove `isTest()` function as well?

Reply:
Removed